web/src/components/molecules/Integration/types.ts

```diff
@@ -0,0 +1,51 @@
+export type IntegrationMember = {
+  integration: Integration;
+  integrationRole: Role;
+  invitedById: string;
+  active: boolean;
+};
+
+export type Role = "WRITER" | "READER" | "OWNER";
+
+export type Integration = {
+  id: string;
+  name: string;
+  description?: string | null;
+  logoUrl: string;
+  developerId: string;
+  developer: Developer;
+  iType: IntegrationType;
+  config: {
+    token?: string;
+    webhooks?: Webhook[];
+  };
+};
+
+export type Developer = {
+  id: string;
+  name: string;
+  email: string;
+};
+
+export enum IntegrationType {
+  Private = "Private",
+  Public = "Public",
+}
```

Comment:
```suggestion
export type IntegrationType = "Private" | "Public"
```
maybe ?
If you do this change please don't forget to replace
`IntegrationType.Private` with `"Private"` and 
`IntegrationType.Public` with `"Public"`

Reply:
I agree :+1: